### ISSUE TITLE
Corrections to Sweater Rubbing Alerts

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#324]
Tested latest build from phettest on Nov 8, 2017, and found some mistakes in the sweater rubbing alerts:

**Sequencing of Sweater rubbing descriptions while picking up charges**
- [x] When rubbing on sweater and picking up charges, I am only hearing alerts 1-1 and 1-5, and not Alert 1-3: "Yellow balloon picks up more negative charges."

Ideally, user would hear Alert 1-3 after the first full description of the charge amount (after the first 1-5 alert) until the charge amount crosses threshold from "a few more" to "several more". And then again after one mention of "several more" I would hear alert 1-3 again, until the charges become "many more". Each visit to sweater is treated like a new visit. First charge pick-up is alert 1-2, second charge pick-up is alert 1-5, and in between thresholds the user gets 1-3 or appropriate alert for picking up last charge (alert 2-2), or finding more pairs (2-1), or if unproductive rubbing on a depleted sweater is happening (alert 2-3).

I hope that is clear and doable.

**Correction needed to Alert 2-3 Hint to leg go (Show all charges view)**
- [x] When rubbing on depleted sweater continues I am hearing:  
  "No change in charges. On right side of sweater. **Sweater has neutral net charge (incorrect)** (and missing period) **Yellow Balloon has many more negative charges than positive charges (wrong string)** (add missing period) Press space to release (capital needed on Space)."
  
  Should hear:
  "No change in charges. On right side of sweater. **Sweater has positive net charge**. **Yellow Balloon has negative net charge.** Press Space to release."

**Correction needed to Alert 2-1 (Show charge differences view)**
- [x] Wrong string and typo. Hearing:
  "No change in **charges**. On right side of sweater. More hidden pairs of charges **string** up and to the left."
 
  Should hear:
  "No change in **net charge**. On right side of sweater. More hidden pairs of charges up and to the left."

**More Punctuation: Extra periods (low priority)**
- [x] Three extra periods in sweater rubbing alert 1-5 (Show diffs) : "Yellow balloon has negative net charge, showing many negative charges..Sweater has positive net charge, showing many positive charges..."

**Comment by jessegreenberg** [comment]
Thanks for the report @terracoda! These issues are fixed in master.
